src/helpers/waitForPerfDataToStabilize.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * This function is used to compare startup performance data and determine if it is stable.
+ * The caller provides a callback that is used to fetch the current value of a string (perf data) and
+ * compares it to the last value.  If the values are the same, the perf data is considered stable.
+ *
+ * @param fetchContent
+ * @param stableData
+ * @param timeout
+ * @param interval
+ * @param fetchContent
+ * @returns
+ */
+export const waitForPerfDataToStabilize = async (
```

Comment:
This is the most troublesome code because:
- uses a hardcoded timeout, which may not be long enough on Windows
- Not sure if the definition of stable is concise enough. Should it be narrowed to establish stability within the 'Extension Activation Stats' section. Is there some other sort of reference that can be used in combination, such as Running Extension list from API call?